### PR TITLE
Handle multiple eslint config formats

### PR DESCRIPTION
**Pull request by adekbadek**
Eslint can be configured using `.eslintrc`/`.eslintrc.js`/`.eslintrc.json` file names, so let's handle them

### CODE REVIEW

**Comment by kasparsd** [comment]
This is great, thank you so much @adekbadek!

File: scripts/check-diff.sh [hunk 1]
```diff
@@ -185,7 +185,7 @@ function set_environment_variables {
 	fi
 
 	if [ -z "$ESLINT_CONFIG" ]; then
-		ESLINT_CONFIG="$( upsearch .eslintrc )"
+		ESLINT_CONFIG="$( upsearch .eslintrc* )"
```
Comment (kasparsd) [inline scripts/check-diff.sh]: @adekbadek Were you able to verify that the wildcard selector actually works with `upsearch`?

Comment (adekbadek) [reply scripts/check-diff.sh]: I checked locally with `.eslintrc` and `.eslintrc.js`, and both worked

**Comment by adekbadek** [comment]
@kasparsd Thanks for merging that! When are you planning a new release?

**Comment by kasparsd** [comment]
@adekbadek Done! https://github.com/xwp/wp-dev-lib/releases/tag/1.4.0

Thanks again for your contribution!

**Pull request merged**